### PR TITLE
chore: convenience Makefile, ignore local tmp dir for scratch files

### PR DESCRIPTION
**Pull request by trentm**
Also anchor many gitignore'd files to the top project dir with '/'.

### CODE REVIEW

**Comment by trentm** [comment]
@astorm Thoughts on this?

1. The Makefile for quick `make check` etc. for common tasks. I can understand hesistance given they are mostly just shims on a single `npm ...` command. Personally my habit of `make check` I'm find hard to break -- esp. to help me catch the commitlint thing I tripping over a lot. :)

2. I'd like to add some top-level tmp dir to gitignore, or some other technique to avoid `standard` trying to apply itself to local scratch/play files. Without it, I get things like:

```
npm run lint

> elastic-apm-node@3.7.0 lint /Users/trentm/el/apm-agent-nodejs
> standard

standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:26:1: Expected indentation of 2 spaces but found 0.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:26:3: Extra semicolon.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:28:1: More than 1 blank line not allowed.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:29:1: Expected { after 'if' condition.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:29:5: Unexpected constant condition.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:30:1: Expected indentation of 2 spaces but found 0.
  /Users/trentm/el/apm-agent-nodejs/tmp/foo.js:31:1: Expected indentation of 4 spaces but found 2.
...
```

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # Files to ignore
 .DS_Store
-.vscode
-package-lock.json
-test/babel/out.js
-test/types/transpile/index.js
+/.vscode
```
Comment (astorm) [inline .gitignore:3]: +1 -- the `/` as a _project root_ anchor is new to mean -- so TIL, etc.

**Review by astorm — APPROVED** [review]
Make's not my favorite, but I'm game -- one more layer of build abstraction isn't going to hurt, and if it helps individual team members be more effective it's a win.  Approved with a "lets try to keep the creeping complexity inherent in `make` in check" caveat :)

**Comment by trentm** [comment]
> one more layer of build abstraction isn't going to hurt

Ouch. Not a thing I'll be proud of then. :)

**Pull request merged**